### PR TITLE
Add unit test for issue 2580

### PR DESCRIPTION
**Pull request by sejongoh**
- The test checks if code generation for ushort is correct.
- #2580 is open for this test.

### CODE REVIEW

**Comment by sejongoh** [comment]
@RussKeldorph This is the same test that I opened a PR recently for ushort comparison. I changed the test name to github_2580.

**Comment by RussKeldorph** [comment]
LGTM

**Comment by sejongoh** [comment]
@dotnet-bot test Ubuntu x64 Checked Build and Test

File: tests/src/JIT/Regression/JitBlue/GitHub_2580/GitHub_2580.cs [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+// setg     cl                       ; set if greater (signed comparison) <- signed comparison for unsigned type                           
+//                        
+// After fix:                                                                                        
+//                         
+// movzx    rcx, word  ptr [rdi+8] ; load a                                                          
+// movzx    rax, word  ptr [rsi+8] ; load b                                                          
+// cmp      cx, ax                 ; compare only 16 bits                                            
+// seta     cl                     ; set if above (unsigned comparison)  <-- Fixed instruction       
+
+internal static class MicrosftNETBug
```
Comment (jasonwilliams200OK) [inline tests/src/JIT/Regression/JitBlue/GitHub_2580/GitHub_2580.cs]: nit: MicrosftNETBug -> MicrosoftNetBug

**Pull request merged**